sample-apps/endoscopy/README.md

```diff
@@ -109,6 +106,11 @@ monailabel start_server \
   --conf epistemic_top_k 3 \
   --conf auto_finetune_models tooltracking \
   --conf auto_finetune_check_interval 30
+
+# Active Learning for Inference
+# After fine-tuned model is saved, push the model to nuclio function container
+/workspace-apps/endoscopy/pushActiveLearningModel.sh
```

Comment:
I think should add the CLI command? For example:
`bash /workspace-apps/endoscopy/pushActiveLearningModel.sh`

Thanks.

Reply:
Thanks